api/envoy/extensions/filters/network/redis_proxy/v3/redis_proxy.proto

```diff
@@ -207,6 +207,34 @@ message RedisProxy {
     repeated string commands = 4;
   }
 
+  message FeatureConfig {
+    // [#next-free-field: 6]
+    message HotKey {
+      enum CacheType { LFU = 0; }
```

Comment:
add comment for each enum and fields please.

Reply:
done